src/test/java/com/hongliangjie/fugue/topicmodeling/LDA/LDATest.java

```diff
@@ -0,0 +1,177 @@
+package com.hongliangjie.fugue.topicmodeling.LDA;
+
+import com.hongliangjie.fugue.Message;
+import com.hongliangjie.fugue.io.DataReader;
+import com.hongliangjie.fugue.serialization.Document;
+import com.hongliangjie.fugue.serialization.Feature;
+import com.hongliangjie.fugue.utils.RandomUtils;
+import org.junit.Test;
+
+import java.net.URL;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by liangjie on 3/6/16.
+ */
+public class LDATest {
+
+    private class DeepTestLDA extends LDA{
+
+        protected class TestProcesser extends ProcessDocuments{
+            Sampler sampler2;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/hongliangjie/Fugue/file/2178366275/issues/source?bid=3162994&fileBranchId=3163615#l23)